app/scripts/modules/kubernetes/pipeline/stages/runJob/runJobStage.html

```diff
@@ -30,6 +30,17 @@
     </div>
   </stage-config-field>
 
+  <stage-config-field label="DNS Policy">
+    <div>
+      <select class="form-control input-sm"
+                ng-model="ctrl.stage.dnsPolicy">
+          <option ng-repeat="policy in ctrl.policies"
```

Comment:
There's another angular syntax for `<select>`s that's a little bit cleaner (I think) and lets you omit `<option>`: https://github.com/spinnaker/deck/blob/master/app/scripts/modules/amazon/src/serverGroup/configure/wizard/zones/availabilityZoneSelector.directive.html#L7-L9

But I don't remember the differences between the two and there may be some good reason for the syntax you've used.